### DISCUSSION TITLE
Navigate Daily/Periodic Notes

### DISCUSSION THREAD

**Discussion by ghost** [SkepticMystic/breadcrumbs#179] [Ideas]
@SkepticMystic Have you thought about using date information in the note names to somehow generate next/previous links? I'm not quite clear in my mind, yet, what exactly that would look like. It seems to me there is a lot of structure that could be used for this plugin.

**Comment by SkepticMystic** [comment]
Ooohhh this is an interesting idea... I could pretty easily make every single note link to the next one based on date created. But this seems impractical. I feel like there should be a way to limit which notes it gets applied to. @joelmeyerhamme What do you think?

**Reply by ghost** [reply]
Maybe have a setting for one or more directories, then link them up by parsing the file names for dates. `moment.js` can guess the format, so that should be good enough to start with.

I'd have to get back into it but I have some more Ideas I could try; it's just that I'm not very comfortable messing with your graph structure. Does that answer the question?

**Comment by michaelpporter** [comment]
In Breadcrumbs v4 this is now built in via the **Date Note** edge builder.

Enable it under **Settings → Date Notes** and set it to match your daily/periodic note format. Breadcrumbs then automatically adds `next`/`prev` edges between chronologically adjacent date notes, so you can jump between them with the Prev/Next view, the trail, or the "Jump to first neighbour" command.

This covers the original request without forcing every note in the vault to chain together — only notes matching your date format are linked.

Closing as resolved — if your date format isn't being picked up, open a fresh issue with the format string and an example filename. 🍞
